service/src/main/java/io/camunda/service/JobServices.java

```diff
@@ -97,8 +97,8 @@ public CompletableFuture<JobRecord> completeJob(
   }
 
   public CompletableFuture<JobRecord> updateJob(
-      final long jobKey, final Integer retries, final Long timeout) {
-    return sendBrokerRequest(new BrokerUpdateJobRequest(jobKey, retries, timeout));
+      final long jobKey, final Map<String, Number> changeset) {
```

Comment:
💭 Changes might also be any arbitrary value in the future, not just number attributes. We don't have anything else so far and the service API is not considered public API, I think. So it should also be fine like this. Would we have any big downsides to using `Map<String, Object> changeset` here instead? Just wondering.